### PR TITLE
Bump the webhook version to 0.4.0-rc3

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

**Review by MbolotSuse — DISMISSED** [review]

**Review by tomleb — APPROVED** [review]

File: tests/framework/extensions/users/users.go [hunk 1]
```diff
@@ -313,7 +313,7 @@ func waitForRTBRollout(client *rancher.Client, rtbNamespace string, rtbName stri
 		Duration: 500 * time.Millisecond,
 		Factor:   1,
 		Jitter:   0,
-		Steps:    11,
+		Steps:    41,
```
Comment (tomleb) [inline tests/framework/extensions/users/users.go:316]: The comment above can be updated. It currently says:

> we expect rollout to happen within 5 seconds total

but now we're waiting for ~20 seconds.

**Review by tomleb — CHANGES_REQUESTED** [review]
Just that small comment update otherwise it's good.

File: tests/framework/extensions/users/users.go (hunk 1 above)
Comment (maxsokolovsky) [reply tests/framework/extensions/users/users.go:316]: Fixed!

**Review by tomleb — APPROVED** [review]
Approved. I wonder though if we should comment why 20 seconds is necessary

**Comment by maxsokolovsky** [comment]
@tomleb, I will add a link to this timeout-changing commit to the issue intending to introduce a long-term fix.

**Review by MbolotSuse — APPROVED** [review]
Approved pending CI pass.

Given that the test changes are (hopefully) not permanent, and only inside test code, I think that this works for a stop-gap measure.

**Review by tomleb — APPROVED** [review]

**Review by MbolotSuse — APPROVED** [review]

**Pull request merged**